src/main/java/io/strimzi/utils/LogicalKafkaVersionEntity.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.JsonNode;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ *  A logical Kafka version
+ */
+public class LogicalKafkaVersionEntity {
+
+    private static final Logger LOGGER = LogManager.getLogger(LogicalKafkaVersionEntity.class);
+
+    private static final Pattern STRIMZI_TEST_CONTAINER_IMAGE_WITHOUT_KAFKA_VERSION = Pattern.compile("^test-container:(\\d+\\.\\d+\\.\\d+|latest)-kafka-.*$");
+    private static final String KAFKA_VERSIONS_URL_JSON = "https://raw.githubusercontent.com/strimzi/test-container-images/main/kafka_versions.yaml"; // this would be replaced by test-container-images url where json will be stored
```

Comment:
Shouldn't this point to a branch, rather than main?

Is there value (e.g. testing?) in allowing a system property to override the URL?

Reply:
I think it's enough to point the `main` explicitly and when we make a release in `test-container-images` in  `release-0.26.x` branch we change to `0.26.x`.